release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -1970,6 +1964,8 @@ sourceStrategy:
 
 * Previously, the secondary scheduler deployment was not deleted after a secondary scheduler custom resource was deleted. Consequently, the Secondary Schedule Operator and Operand were not fully uninstalled. With this update, the correct owner reference is set in the secondary scheduler custom resource so that it points to the secondary scheduler deployment. As a result, secondary scheduler deployments are deleted when the secondary scheduler custom resource is deleted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2100923[*BZ#2100923*])
 
+* For the {product-title} {product-version} release, the descheduler can now publish events to an API group, because the release adds additional role-based access controls (RBAC) rules to the descheduler's profile.(link:https://issues.redhat.com/browse/OCPBUGS-2330[*OCPBUGS-2330*])
```

Comment:
```suggestion
* For the {product-title} {product-version} release, the descheduler can now publish events to an API group because the release adds additional role-based access controls (RBAC) rules to the descheduler's profile.(link:https://issues.redhat.com/browse/OCPBUGS-2330[*OCPBUGS-2330*])
```